### DISCUSSION TITLE
one table of contents file for in-network files and a separate one for allowed amounts files

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#760] [Q&A]
Is it valid to have one table of contents file for in-network files and a separate one for allowed amounts files for the same plans?
There are payers that are now posting separate index files, one for in_network files and one for allowed_amounts files.

While the examples on GitHub show a single table of contents covering a set of plans and creating the relationships between plans and files, the language on GitHub doesn't directly address this issue.

If this were allowed then the naming convention for index files would have to be extended since now it says:

> The following is the required naming standard for the table-of-contents file: <YYYY-MM-DD>_<payer or issuer name>_index.<file extension>

And that convention doesn't seem to permit two table of contents files for the same payer/issuer.

It would also introduce other issues e.g. what would be the key for matching plans between index files?

**Comment by shaselton-usds** [comment]
@nmayle Thanks for the heads up.

You are correct, the allowed amount and in-network files should be grouped together for the single plan in a single table of contents file. How often are you seeing this?

If you wish to submit a complaint, [you can do so here](https://mats-cms-ccrms.my.salesforce-sites.com/pt).

**Comment by nmayle** [comment]
Scott,

I did some querying of our data and I wasn’t able to find other occurrences of this issue, but I do believe I have seen them.
It may not be a common “mistake/misunderstanding”.  The reason I wrote about it is that when I re-read the language on Github I realized that there was nothing explicitly saying that this was not how the index files should be used.

The payer that I did locate has quite a few problems including how they have named their files:

https://www.mclarenhealthplan.org/mhp/transparency-in-coverage-and-no-surprises-act

* table_of_content vs index
* separate index files for in_network and allowed_amounts

If payers don’t follow the naming convention for index files, it can be hard for us to identify index files without downloading them and looking at the contents.
